code/game/turfs/simulated/walls_mineral.dm

```diff
@@ -117,9 +117,9 @@
 	..()
 
 /turf/simulated/wall/mineral/plasma/proc/PlasmaBurn(temperature)
-	var/pdiff=performWallPressureCheck(src.loc)
-	if(pdiff>0)
-		message_admins("Plasma wall with pdiff [pdiff] at [formatJumpTo(loc)] just caught fire!")
+	var/pdiff = performWallPressureCheck(src.loc)
+	if(pdiff > 0)
+		investigation_log(I_ATMOS, "with a pdiff of [pdiff] caught on fire at [formatJumpTo(get_turf(src))]!")
```

Comment:
The name of the item being logged is presented before the rest of the message, so this would say "the plasma wall [message]"